app/src/components/BackButton/index.tsx

```diff
@@ -1,12 +1,20 @@
-import * as React from 'react';
 import useDispatch from 'hooks/useDispatch';
 import styles from './styles.module.css';
 import { ReactComponent as CloseIcon } from 'icons/close.svg';
 import { openTask } from 'modules/selectedScreen';
+import useEventListener from 'hooks/useEventListener';
+import keyCodes from 'utils/keyCodes';
 
 export default function BackButton() {
   const dispatch = useDispatch();
 
+  useEventListener('keydown', (e) => {
+    if(e.keyCode === keyCodes.esc) {
+      dispatch(openTask())
+    }
+  })
```

Comment:
Lets have this shortcut be in `app/src/hooks/useShortcuts`, so all global shortcuts are in one place.